src/Heading/Heading.js

```diff
@@ -119,7 +120,7 @@ Heading.propTypes = {
    * Optional - to override the default size for the heading level selected.
    * Step 0 = base font size; each step up is a factor of 1.25 of the previous.
    */
-  size: PropTypes.oneOf(Object.keys(headingSizes)),
+  size: PropTypes.oneOf(Object.keys(headingSizes).map((val) => intKeys(val))),
```

Comment:
Does the heading level need this treatment too or is that one fine as is?

Reply:
The level one doesn't because the levels are in an array and an array can have integer values. The problem with using the object keys is that they are always cast to strings.

Reply:
Ah, good to know. Thanks! GTG